api/config/config_local.json

```diff
@@ -97,7 +97,8 @@
       "expiryDays": 365,
       "expiryDaysWarningThresholds": [1, 3, 7, 15, 30],
       "lookbackPeriod": 330
-    }
+    },
+    "tiersVisibleToUsers": ["registered", "controlled"]
```

Comment:
These all match the UI's `environment.accessTiersVisibleToUsers` (a PR to merge these will be coming soon)

Stable and Prod: Registered only
All others: Registered and Controlled